#!/usr/bin/env python
PACKAGE = "wpm2_tutorials"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Joint_1", int_t , 0 , "Joint_1", 0 , -180 , 180)
gen.add("Joint_2", int_t , 0 , "Joint_2", 0 , -180 , 180)
gen.add("Joint_3", int_t , 0 , "Joint_3", 0 , -180 , 180)
gen.add("Joint_4", int_t , 0 , "Joint_4", 0 , -180 , 180)
gen.add("Joint_5", int_t , 0 , "Joint_5", 0 , -180 , 180)
gen.add("Joint_6", int_t , 0 , "Joint_6", 0 , -180 , 180)
gen.add("Gripper", int_t , 0 , "Gripper", 0 , 10000 , 35000)

exit(gen.generate(PACKAGE, "wpm2_joint_rcfg", "JointDegree"))